### PR TITLE
update FAQs

### PR DESCRIPTION
**Pull request by dmitrydao**

### CODE REVIEW

File: _data/navigation.js [hunk 1]
```diff
@@ -140,6 +140,13 @@ module.exports = {
           },
         ],
       },
+      {
+        section: 'ADDITIONAL INFORMATION',
+        contents: [
+          { url: '/docs/node-operators-faq/', title: 'FAQ' },
+          { url: '/docs/glossary/', title: 'Glossary' },
```
Comment (dwightjl) [inline _data/navigation.js:147]: Let's call this page "Terminology" instead. Glossary is also accurate, but I think Terminology is more clear.

File: _data/navigation.js [hunk 2]
```diff
@@ -140,6 +140,13 @@ module.exports = {
           },
         ],
       },
+      {
+        section: 'ADDITIONAL INFORMATION',
+        contents: [
+          { url: '/docs/node-operators-faq/', title: 'FAQ' },
```
Comment (dwightjl) [inline _data/navigation.js:146]: Let's leave out the FAQ for now. There are some questions like "what is a phase" that should be answered in the Glossary/Terminology page though.

File: _data/navigation.js [hunk 3]
```diff
@@ -140,6 +140,13 @@ module.exports = {
           },
         ],
       },
+      {
+        section: 'ADDITIONAL INFORMATION',
```
Comment (dwightjl) [inline _data/navigation.js:144]: Let's leave the Glossary/Terminology page in the Developer Reference for now.

File: docs/Additional Information/contract-devs-faq.md [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+
+You can get multiple values by using the [Multi-Variable Responses](/docs/multi-variable-responses/) feature.
+
+## Price Feeds FAQ
+
+### Why don’t the rounds update chronologically?
+
+They do, in the best-case scenario. However, a round can time out if it doesn’t reach consensus, so that would technically be a timed out round, which carries over the answer from the previous round. Though roundIds can seemingly jump significantly when the phaseId is updated, because of how that combination of phaseId+roundId is calculated by the proxy.
+
+### What is a phase?
```
Comment (dwightjl) [inline docs/Additional Information/contract-devs-faq.md:63]: Move this to the Glossary.

**Review by dwightjl — CHANGES_REQUESTED** [review]

File: docs/Additional Information/contract-devs-faq.md (hunk 4 above)
Comment (dmitrydao) [reply docs/Additional Information/contract-devs-faq.md:63]: Not sure. Patrick mentioned this is very frequent question so I think we can leave it here.

File: _data/navigation.js (hunk 1 above)
Comment (dmitrydao) [reply _data/navigation.js:147]: We had a glossary before. It's very common in most blockchain documentation so people got used to looking for Glossary.

File: docs/Additional Information/contract-devs-faq.md [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+layout: nodes.liquid
+date: Last Modified
+title: "Contract Developers FAQ"
+permalink: "docs/contract-developers-faq/"
+metadata: 
+  title: "Contract Developer FAQ"
+  description: "Find answers to common questions about developing on Chainlink."
+---
+
+## General FAQ
```
Comment (dwightjl) [inline docs/Additional Information/contract-devs-faq.md:11]: Can we take the information in this FAQ and place it in the related documents in non-FAQ format? I think many of these questions have associated documentation where they belong.

**Review by dwightjl — CHANGES_REQUESTED** [review]

File: _includes/main.liquid [hunk 6]
```diff
@@ -58,6 +58,9 @@
             })(window,document,'script','dataLayer','GTM-N6DQ47T');</script>
         <!-- End Google Tag Manager -->
 
+        {% if page.url == "/docs/contract-developers-faq/" %}
```
Comment (dwightjl) [inline _includes/main.liquid:61]: Add comments to explain what these if statements do.

**Review by dwightjl — CHANGES_REQUESTED** [review]

File: docs/Additional Information/contract-devs-faq.md (hunk 5 above)
Comment (dmitrydao) [reply docs/Additional Information/contract-devs-faq.md:11]: Not sure what other document do you want this to go to. This is a fairly regular question people have been asking for and still continue to ask for some reason.

Comment (dwightjl) [reply docs/Additional Information/contract-devs-faq.md:11]: I think I know where I want to reorganize this information. Give me some time to go through it and I'll put together a draft.

**Comment by dwightjl** [comment]
Closing this in favor of https://github.com/smartcontractkit/documentation/pull/624

**Pull request closed without merging**